opencti-platform/opencti-front/src/schema/relay.schema.graphql

```diff
@@ -1244,7 +1244,8 @@ type User implements BasicObject & InternalObject {
   theme: String
   language: String
   external: Boolean
-  dashboard: String
+  dashboard_id: ID
+  dashboard: Workspace
```

Comment:
I'm wondering why do we need to expose `dashboard_id` and `dashboard`, can't we just use `dashboard` ?

Reply:
We need it because we will have a bit of logic with those dashboards : 
- Show in admin what dashoard a user has as default
- Compute at login a list of default dashboards for a user (coming from groups and orgas)

So dashboard_id will still be the "default dashboard choosen by user" and dashboard will be dashboards: [Workspace] representing the list of default dashboards assigned to him through groups and orga

Reply:
Ok, I still don't understand why `dashboard_id` and not dashboard, which can get us the `name`. In the admin, we show the name of the default dashboard, we don't just use the id.

Reply:
Dashboard_id = default dashboard choosen by user
Dashboard = default dashboard for user (will be computed from other stuff)

Reply:
A user can have two different default dashboards then ?

Reply:
Nop. He will have a list of default dashboards defined by its organization and groups and only one **active** default dashboard that will be either the first one of the list or the one stored in dashboard_id because he has choosen that one

Reply:
> I can see in user resolver :
> 
> ```
> dashboard: (current, _, context) => {
>       return current.dashboard_id ? findWorskpaceById(context, context.user, current.dashboard_id) : null;
> ```
> 
> So dashboard and dashboard_id are the same.

Yes because we don't have groups/orgas yet

Reply:
wow, this is really not easy to understand by just looking at the code

Reply:
I still don't really get why in the `me` query, we get `dashboard_id` instead of `dashboard { id }`

Reply:
> I still don't really get why in the `me` query, we get `dashboard_id` instead of `dashboard { id }`

Yep, we could do that. But in anycase it will be modified in next step.